Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py

```diff
@@ -0,0 +1,795 @@
+            incidents = []
+
+        now = datetime.utcnow()
+        start_time = now.strftime("%s")
+        demisto.info("FrontlineVM fetch_incident -- creating new start_time of " + str(now.strftime("%Y-%m-%dT%H:%M:%S.%fZ")))
+        demisto.setLastRun({'start_time': start_time})
+        demisto.incidents(incidents)
+    except Exception as err:
+        return_error("Error: FrontlineVM fetching_incidents -- " + str(err))
+
+
+def get_hosts(ip_address, hostname, label_name, max_days_since_scan):
+    ''' Returns a list of hosts from Frontline.Cloud based on user input from Arguments    '''
+    # Prepare parameters for Frontline API request:
+    req_params = {}
+    if ip_address:
+        req_params['eq_host_ip_address'] = str(ip_address)
+    if hostname:
+        req_params['iexact_host_hostname'] = str(hostname)
+    if label_name:
+        req_params['eq_host_labels'] = str(label_name)
+    if max_days_since_scan:
+        try:
+            query_date = get_query_date_param(max_days_since_scan)
+            req_params['gte_host_date_created'] = str(query_date)
+        except ValueError:
+            debug_msg = "incorrect data type input for argument max_days_since_scan, should be number of days"
+            demisto.debug("FrontlineVM get_hosts -- " + debug_msg)
+            return_error("Error: max_days_since_scan value should be a number representing days.")
+    req_params = parse_params(req_params)
+    hosts = get_fvm_data(HOST_ENDPOINT, params=req_params)
+    return hosts
+
+
+def get_assets_command():
+    ''' Pulls host information from Frontline.Cloud '''
+    # Get Arguments:
+    ip_address = demisto.args().get('ip_address')
+    hostname = demisto.args().get('hostname')
+    label_name = demisto.args().get('label_name')
+    max_days_since_scan = demisto.args().get('max_days_since_scan')
+
+    hosts = get_hosts(ip_address, hostname, label_name, max_days_since_scan)
+
+    # Condensing Host data for HumanReadable and EntryContext:
+    asset_output = {}
+    host_list = []
+    ip_list = []
+    host_id_list = []
+    for host in hosts:
+        host_obj = {}
+        host_obj['ID'] = host.get('id', None)
+        host_obj['Hostname'] = host.get('hostname', '')
+        host_obj['IP'] = host.get('ip_address', '')
+        host_obj['DNSHostname'] = host.get('dns_name', '')
+        host_obj['MAC'] = host.get('mac_address', '')
+        host_obj['OS'] = host.get('os')
+        host_obj['OSType'] = host.get('os_type')
+        host_obj['CriticalVulnCount'] = host['active_view_vulnerability_severity_counts']['weighted']['ddi']['counts']['critical']
+        host_list.append(host_obj)
+        ip_list.append(host.get('ip_address'))
+        host_id_list.append(host.get('id'))
+
+    # Linking Context:
+    asset_output['Hosts'] = host_list
+    asset_output['IPList'] = ip_list
+
+    entry_context = {'FrontlineVM(val.Hosts && val.Hosts == obj.Hosts)': asset_output}
```

Comment:
in my opinion you should return this object, and build the output in the function calling this one

Reply:
Nothing is calling this method except the command. This is the main `frontline-get-assets` command method. 

Changed to include pass whole output object into `demisto.return()` instead of just `demsito.return(object)`.